docs/jwst/tweakreg/index.rst

```diff
@@ -1,13 +1,25 @@
 ========
-Tweakreg
+tweakreg
 ========
 
+.. moduleauthor:: Mihai Cara <help@stsci.edu>
+
 .. toctree::
    :maxdepth: 2
 
-   description.rst
-   wcsimage.rst
-   wcsutils.rst
-   linearfit.rst
+   README.rst
+
+**Also See:**
+
+.. toctree::
+   :maxdepth: 2
+
+   imalign
+   wcsimage
+   linearfit
+   matchutils
+   tweakreg_catalog
```

Comment:
I don't think tweakreg_catalog and tweakreg_step exist anymore. They've been combined into the single tweakreg, which is what these docs are about. Or do they still exist as lower-level functions/modules?

Reply:
@hbushouse I see both files in https://github.com/STScI-JWST/jwst/tree/master/jwst/tweakreg The new `tweakreg` step essentially combines the functionality of the two steps (old `tweakreg` and `tweakreg_catalog` steps). Probably `tweakreg_catalog.py` holds what was not moved to the common `tweakreg_step.py`. Maybe `tweakreg_catalog.py` should be renamed to simply `catalog.py`